Ошибка с более чем одним передаваемым аргументов в обработчик связана с использованием имени callback_data в качестве первого
принимаемого аргумента